styles/google-gemini/catppuccin.user.css

```diff
@@ -221,40 +216,36 @@
 
     --gem-sys-color--surface: @base;
     --gem-sys-color--surface-container: @mantle;
+    --gem-sys-color--surface--rgb: #rgbify(@base) [];
     --gem-sys-color--on-surface: @text;
+    --gem-sys-color--on-surface-variant: @subtext1;
     --gem-sys-color--outline-variant: @subtext0;
     --gem-sys-color--surface-container-high: @surface0;
     --gem-sys-color--surface-container-low: lighten(@mantle, 2%);
     --gem-sys-color--outline: @subtext0;
-    --gem-sys-color--on-surface-variant: @subtext1;
     --gem-sys-color--on-primary-container: @text;
+    --gem-sys-color--on-primary-fixed: darken(@accent-color, 40%);
+    --gem-sys-color--on-primary: darken(@accent-color, 20%);
```

Comment:
This looks very out of place:

![CleanShot 2024-10-28 at 09 09 36](https://github.com/user-attachments/assets/63661641-63da-4ffa-aeec-346069313c63)

What was this intended to theme? I would suggest using just regular accent.

```suggestion
    --gem-sys-color--on-primary: @accent-color;
```

Reply:
Also, in other places this breaks things. `--on-primary` implies this is the text color of elements with a background of primary? For example, a button in the "about your data" popup you get after clicking upload image button is unreadable since both the text color and the background color are primary.

Reply:
I am trying to follow official style as closely as possible, officially its darkened, so i done the same thing here. using plain accent-color only breaks things.

i dont understand what are you trying to say, can you explain more.

Reply:
> I am trying to follow official style as closely as possible, officially its darkened, so i done the same thing here. using plain accent-color only breaks things.

Actually in most cases we aren't trying to copy exactly what shades of colors are used. When theming a website for a userstyle you typically will emulate the general styling (e.g. bordered buttons, change on hover, whatever) but that doesn't mean darkening colors this much. Especially since beyond 10-15% it no longer resembles the color.

> 
> i dont understand what are you trying to say, can you explain more.

Sorry, I think I got a bit confused with these variables. I think you might have to do more specific scoped styling since just the variables alone don''t work well in every case.